src/components/Layer.js

```diff
@@ -177,7 +177,7 @@ export class Layer extends Component {
     if (relationshipLabel != null && relationshipLabel.length > 0) {
       let sub_q = "(false";
       relationshipLabel.forEach((value) => {
-        sub_q += ` OR r:\`${value.label}\``;
+        sub_q += ` OR type(r) = '${value.label}' `;
```

Comment:
i tested it and it was working for me :thinking: 
maybe it depends on cypher/neo4j version, i'll double check this

Reply:
I was surprised too because it seemed to me I had already tested it, but tested in a Cypher shell and raises the same error. Documentation here: https://neo4j.com/docs/cypher-manual/4.3/clauses/where/#filter-on-relationship-type, only mention the "type(r)" method, so I think it is safer to go this way.